app/pages/patientdata/patientdata.less

```diff
@@ -18,33 +18,51 @@
   position: relative;
 }
 
+// Header
+// ====================================
+
+.nav-patient-header {
+  margin: 0 auto 16px;
+  width: 100%;
+  max-width: @screen-lg-min;
+
+  @media screen and (max-width: @screen-lg-min) {
+    padding: 0 20px;
+  }
+}
+
 // Subnav
 // ====================================
 
 @patient-data-subnav-vertical-padding: @spacing-small;
-@patient-data-subnav-horizontal-padding: @spacing-small;
+@patient-data-subnav-horizontal-padding: @spacing-medium;
+
+@patient-data-subnav-breakpoint: 1080px;
```

Comment:
I don't think this belongs in the global screen variables, but this breakpoint allows the header to stay in the "normal" configuration until it's absolutely necessary to switch to a column-type configuration.

https://github.com/user-attachments/assets/29dd1804-9539-4209-855f-abdb89f03bb8